src/main/java/by/onliner/page/components/ShoppingCartPopup.java

```diff
@@ -0,0 +1,43 @@
+package by.onliner.page.components;
+
+import by.onliner.page.AbstractComponent;
+import by.onliner.util.Waiter;
+import org.awaitility.Awaitility;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import static java.util.concurrent.TimeUnit.SECONDS;
+
+public class ShoppingCartPopup extends AbstractComponent {
+
+    @FindBy(className = "product-recommended__sidebar-close")
+    private WebElement shoppingCartPopupCloseButton;
+
+    public ShoppingCartPopup() {
+        super();
+    }
+
+    @Override
+    public void waitForComponentOpened() {
+        Waiter.elementToBeClickable(shoppingCartPopupCloseButton);
+    }
+
+    public boolean visibilityOfShoppingCartPopup() {
+        try {
+            Awaitility.await()
+                    .atMost(5, SECONDS)
```

Comment:
could you please update the formatting? the statements are quite small so i think there is no sense in new lines for each

Reply:
⭐